commercetools-models/src/main/java/io/sphere/sdk/projects/commands/ProjectUpdateCommand.java

```diff
@@ -0,0 +1,41 @@
+package io.sphere.sdk.projects.commands;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.commands.Command;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.projects.Project;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Update commands for the {@link Project} entity
+ *{@include.example io.sphere.sdk.projects.commands.ProjectUpdateActionsIntegrationTest#execution()}
+ */
+@JsonDeserialize(as = ProjectUpdateCommandImpl.class)
```

Comment:
Is this annotation really necessary? We only serialize commands, but never derislize them.

Reply:
👍  good catch, actually its not necessary, so i would better eliminate it to avoid confusion.